docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -1,632 +1,117 @@
 :test-signers: https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.6.1/tests/signers.py
+:snip-5: https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-5.md
+:snip-6: https://github.com/ericnordelo/SNIPs/blob/feat/standard-account/SNIPS/snip-6.md
 
 = Accounts
 
-Unlike Ethereum where accounts are directly derived from a private key, there's no native account concept on StarkNet.
+Unlike Ethereum where accounts are directly derived from a private key, there's no native Externally Owned Account (EOA) concept on Starknet.
 
-Instead, signature validation has to be done at the contract level.
-To relieve smart contract applications such as ERC20 tokens or exchanges from this responsibility, we make use of Account contracts to deal with transaction authentication.
+Instead, the L2 features native account abstraction, and signature validation has to be done at the contract level. To relieve smart contract applications such as ERC20 tokens or exchanges from this responsibility, we make use of Account contracts to deal with transaction authentication.
```

Comment:
>I'd switch this Starknet because there's no ref to L2

I didn't want to use Starknet because the previous sentence (two words before) ends with Starknet. Agree L2 is perhaps not the best approach, what do you think of "Instead, it features..."?

Reply:
>And it's a small pet peeve of mine, but I vote for putting sentences on their own line so editing/referencing isn't so noisy. Same goes for the whole PR

I will update accordingly

Reply:
> I didn't want to use Starknet because the previous sentence (two words before) ends with Starknet. Agree L2 is perhaps not the best approach, what do you think of "Instead, it features..."?

Sounds good to me

Reply:
I used "the network" at last, wdyt?

Reply:
Even better 👍